Samples/excel-data-types-explorer/src/commands/commands.html

```diff
@@ -0,0 +1,18 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. -->
```

Comment:
Is anything in `/commands` used or needed? Or can we remove this vestigial yo office code?